contracts/src/core/BatcherPaymentService.sol

```diff
@@ -223,6 +231,8 @@ contract BatcherPaymentService is
         } else {
             require(layer[0] == batchMerkleRoot, "Invalid merkle root");
         }
+
+        emit MerkleRootVerified(batchMerkleRoot);
```

Comment:
I can't seem to understand the added value of this event

Reply:
It was pointed out by the auditors: 
> The `checkMerkleRootAndVerifySignatures` function, which is public and updates user balances in `verifySignatureAndDecreaseBalance` but does not emit any events.
Maybe it doesn't make sense, we can discuss it

Reply:
We made that function internal, it was a mistake it being public

Reply:
Removed in [86832b2](https://github.com/yetanotherco/aligned_layer/pull/840/commits/86832b2bd076581b83817d8dacf9eb106e5d087d)